target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-deco-m5-v2.dts

```diff
@@ -0,0 +1,371 @@
+// SPDX-License-Identifier: ISC
+// Copyright (c) 2015, The Linux Foundation. All rights reserved.
+// Copyright (c) 2021, Frank Veltmans <veltmans.frank@gmail.com>
+ 
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+ / {
+	model = "TP-Link Deco M5";
+	compatible = "tplink,deco-m5", "qcom,ipq4019";
+
+	chosen {
+		bootargs-append = " root=/dev/mtdblock16";
+	};
+
+	aliases {
+		spi0 = "/soc/spi@78b5000";
+		spi1 = "/soc/spi@78b6000";
+		i2c0 = "/soc/i2c@78b7000";
+		i2c1 = "/soc/i2c@78b8000";
+		led-boot = &led_blue;
+		led-failsafe = &led_red;
+		led-running = &led_green;
+		led-upgrade = &led_red;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	soc {
+		interrupt-controller@b000000 {
+			compatible = "qcom,msm-qgic2";
+			interrupt-controller;
+			#interrupt-cells = < 0x03 >;
+			reg = < 0xb000000 0x1000 0xb002000 0x1000 >;
+			linux,phandle = < 0x01 >;
+			phandle = < 0x01 >;
+		};
+
+		pinctrl@1000000 {
```

Comment:
Please use the existing labels whenever they exist and not put it under soc node.